packages/backend/src/api/routers/ForcedTradeOfferRouter.ts

```diff
@@ -0,0 +1,72 @@
+import { AssetId, StarkKey } from '@explorer/types'
+import Router from '@koa/router'
+import bodyParser from 'koa-bodyparser'
+import { string, z } from 'zod'
+
+import { ForcedTradeOfferController } from '../controllers/ForcedTradeOfferController'
+import {
+  stringAs,
+  stringAsBigInt,
+  stringAsInt,
+  withTypedContext,
+} from './types'
+import { applyControllerResult } from './utils'
+
+export function createForcedTradeOfferRouter(
+  offerController: ForcedTradeOfferController
+) {
+  const router = new Router()
+
+  router.post(
+    '/offer',
+    bodyParser(),
+    withTypedContext(
+      z.object({
+        request: z.object({
+          body: z.object({
+            starkKeyA: stringAs(StarkKey),
+            positionIdA: stringAsBigInt(),
+            syntheticAssetId: stringAs(AssetId),
+            amountCollateral: stringAsBigInt(),
+            amountSynthetic: stringAsBigInt(),
+            aIsBuyingSynthetic: z.boolean(),
+          }),
+        }),
+      }),
+      async (ctx) => {
+        const result = await offerController.postOffer(ctx.request.body)
+        applyControllerResult(ctx, result)
+      }
+    )
+  )
+
+  router.post(
+    '/offer/:initialOfferId/accept',
+    bodyParser(),
+    withTypedContext(
+      z.object({
+        params: z.object({
+          initialOfferId: stringAsInt(),
+        }),
+        request: z.object({
+          body: z.object({
+            starkKeyB: stringAs(StarkKey),
+            positionIdB: stringAsBigInt(),
+            submissionExpirationTime: stringAsInt(), // Timestamp?
```

Comment:
Actually I think this is in hours? Please double check with solidity code

Reply:
Yes, seems like https://github.com/starkware-libs/starkex-contracts/blob/master/scalable-dex/contracts/src/perpetual/interactions/ForcedTrades.sol#L57 Interestingly, the [tx](https://etherscan.io/tx/0x9b2dce5538d0c8c08511c9383be9b67da6f952b367baff0c8bdb5f66c9395634) that we use as an example, had an expiration time in 40k years